### ISSUE TITLE
Picking image using Camera does not return FilePath

### ISSUE THREAD

**Issue by Yunus0or1** [Sh1d0w/multi_image_picker#226]
**Bug Report**

`multi_image_picker` version: 4.5.2

Is it happening on Android: Yes

Is it happening on iOS: Not tested

```
[√] Flutter (Channel stable, v1.7.8+hotfix.4, on Microsoft Windows [Version 10.0.18362.239], locale en-IN)
    • Flutter version 1.7.8+hotfix.4 at C:\Flutter
    • Framework revision 20e59316b8 (5 weeks ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\Yunus\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[√] Android Studio (version 3.4)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 38.2.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[√] VS Code (version 1.37.1)
    • VS Code at C:\Users\Yunus\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.3.0

[√] Connected device (1 available)
    • SM G900H • 3208562a41745157 • android-arm • Android 6.0.1 (API 23)

• No issues found!
```

**Describe the bug**

If 'enableCamera: true' and anyone takes image with the Camera, chooses that image file and requests for FilePath using 
`String path = await asset.filePath;` there comes this error
`[ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: PlatformException(ASSET_DOES_NOT_EXIST, The requested image does not exist., null)`

Probable Cause: The image might be in Ram, not in ROM. That is why file not found by that method

**Comment by Renkon117** [comment]
I have the same issue!! please help

**Comment by Sh1d0w** [comment]
I think the problem is in FishBun, when the image is picked it is not refreshed in the media scanner cache, that's why it errors. I need to confirm that when I have time, meanwhile you can use `getByteData` or `getThumbByteData` to obtain the image.

**Comment by fralways** [comment]
getByteData also returns: The requested image does not exist.
multi_image_picker: ^4.5.2

**Comment by dheeraj2dj** [comment]
same issue with me!

**Comment by Sh1d0w** [comment]
The issue is related to the FishBun plugin, opened a bug report there https://github.com/sangcomz/FishBun/issues/167

Once it gets fixed I will update the repo.

**Comment by dheeraj2dj** [comment]
@Sh1d0w This is breaking my production app. Any suggestion for a workaround?

**Comment by Sh1d0w** [comment]
@dheeraj2dj Unfortunately since the bug is in a third party plugin I am using I can't do much.

You can disable the camera feature on Android for the time being.

**Comment by dheeraj2dj** [comment]
@Sh1d0w Any particular version to which I can downgrade to? May be `4.5.0+2` - as I started with that

**Comment by Sh1d0w** [comment]
I don't believe it was working there either, but you can try. It is using the same FishBun version.

Can you check and confirm it was working there?

**Comment by dheeraj2dj** [comment]
@Sh1d0w Surprisingly, that works! I am now on `4.5.0+2`.

**Comment by Sh1d0w** [comment]
If it worked in previous versions it maybe caused by introducing more strict check in the latest version here

https://github.com/Sh1d0w/multi_image_picker/commit/1230ef1518659e84f35a28c5ae21d5a3f053ed7a#diff-a1e9afd876d647cc741fa9593d690b03R296

I need to revise if this broke it, maybe the media scanner on android need a few ms to refresh the image after it is picked from the camera before it become available in the case, which previously it had because of the async task.

**Comment by Sh1d0w** [comment]
This should be fixed in version 4.5.3
